build.xml

```diff
@@ -13,8 +13,8 @@
     <property name="build.dependencies" location="dependencies" />
 
     <path id="classpath">
-        <fileset dir="/usr/share/java" includes="**/*.jar" />
         <fileset dir="${build.dependencies}" includes="**/*.jar" />
+        <fileset dir="/usr/share/java" includes="**/*.jar" />
```

Comment:
Is this the change that fixes https://github.com/awslabs/amazon-kinesis-agent/issues/21?

This is what I did too. Thanks.